src/kbmod/analysis_utils.py

```diff
@@ -54,8 +55,10 @@ def load_images(
         -------
             stack : `kbmod.ImageStack`
                 The stack of images loaded.
-            img_info : `ImageInfo`
-                The information for the images loaded.
+            wcs_list : `list`
+                 A `list` of `astropy.wcs.WCS` objects for each image.
+            visit_times : list
```

Comment:
haha, sorry, I guess I wasn't clear enough. 

```
visit_times : `list`
```

Things that are single back-ticked Sphinx interprets as a reference - in this case to the builtin list type - and things that are double-backticked Sphinx interprets as a literal. So backticking things like `kbmod.ImageStack` craetes a link to the class documentation and things that are double back-ticked are just `grayed_out_literals` like these. So the lists int he explanations don't need to be backticked, the lists in the type description should.

Reply:
Fixed.